packages/fuels-test-helpers/src/lib.rs

```diff
@@ -188,19 +190,26 @@ pub async fn setup_test_client(
     let config_with_coins: Value =
         serde_json::from_str(result.as_str()).expect("Failed to build config_with_coins JSON");
 
+    let transaction_parameters = serde_json::to_value(ConsensusParameters::DEFAULT).expect("Failed to build transaction_parameters JSON");
```

Comment:
Maybe we should limit the proliferation of serde? Keep the data in domain types until the last moment when you actually need them to be serialized.

Reply:
Also, maybe `transaction_parameters` is not ideal as a variable name in this context since `ConsensusParameters` could be confused with `TxParameters`